packages/flutter/test_fixes/widgets.dart

```diff
@@ -184,4 +185,38 @@ void main() {
 
   // Change made in https://github.com/flutter/flutter/pull/100381
   TextSelectionOverlay.fadeDuration;
+
+  // Changes made in https://github.com/flutter/flutter/pull/105291
+  BoxDecoration(color: Color(0xFFFF0000), boxShadow: []);
+  BoxDecoration(
+    shape: BoxShape.rectangle,
+    color: Color(0xFFFF0000),
+    boxShadow: [],
+  );
+  BoxDecoration(
+    color: Color(0xFFFF0000),
```

Comment:
Only code that contains an error or triggers a lint would be evaluated for a fix. `shape` is what is being deprecated, so this code that does not contain `shape` would not trigger a fix.

Reply:
Thanks!!